packages/components/src/dropdown/index.js

```diff
@@ -85,6 +85,7 @@ export default function Dropdown( {
 		<div
 			className={ classnames( 'components-dropdown', className ) }
 			ref={ containerRef }
+			tabIndex="-1"
```

Comment:
Could you leave a comment here? Maybe

```suggestion
			// Safari sometimes moves focus to a focusable parent when clicking
			// a button. By making this div focusable, we can detect if the
			// toggle has been clicked in Safari for `closeIfFocusOutside`.
			tabIndex="-1"
```